src/Raven.Studio/wwwroot/App/views/manage/certificates.html

```diff
@@ -191,10 +191,14 @@ <h5><i class="icon-refresh"></i> AUTO RENEWAL DATE</h5>
                                     </div>
                                     <div class="permissions">
                                         <h5><i class="icon-database"></i> DATABASE PERMISSIONS</h5>
-                                        <div class="flex-horizontal flex-wrap" data-bind="foreach: $root.resolveDatabasesAccess($data)">
-                                            <strong data-bind="text: $data"></strong>
+                                        <div class="flex-horizontal flex-wrap" data-bind="foreach: $root.databasesAccessInfo($data)">
+                                            <div class="margin-right" data-placement="bottom" data-toggle="tooltip" data-animation="true"
+                                                 data-bind="attr: { title: $root.getAccessInfoText(dbName, accessLevel) }">
+                                                <strong data-bind="text: dbName"></strong>
+                                                <i data-bind="attr: { class: $root.getAccessColor(accessLevel)() + ' ' + $root.getAccessClass(accessLevel)() }"></i>
```

Comment:
why we create new observable each time? Looks weird.

Reply:
done - removed the computed from inside the method